vms/avm/index_test.go

```diff
@@ -468,74 +285,28 @@ func buildTX(utxoID avax.UTXOID, txAssetID avax.Asset, address ...ids.ShortID) *
 	}}
 }
 
-func setupTestVM(t *testing.T, ctx *snow.Context, baseDBManager manager.Manager, genesisBytes []byte, issuer chan common.Message, config Config) *VM {
-	require := require.New(t)
-
-	vm := &VM{}
-	avmConfigBytes, err := json.Marshal(config)
-	require.NoError(err)
-	appSender := &common.SenderTest{T: t}
-
-	require.NoError(vm.Initialize(
-		context.Background(),
-		ctx,
-		baseDBManager.NewPrefixDBManager([]byte{1}),
-		genesisBytes,
-		nil,
-		avmConfigBytes,
-		issuer,
-		[]*common.Fx{{
-			ID: ids.Empty,
-			Fx: &secp256k1fx.Fx{},
-		}},
-		appSender,
-	))
-
-	vm.batchTimeout = 0
-
-	require.NoError(vm.SetState(context.Background(), snow.Bootstrapping))
-
-	require.NoError(vm.SetState(context.Background(), snow.NormalOp))
-	return vm
-}
-
 func assertLatestIdx(t *testing.T, db database.Database, sourceAddress ids.ShortID, assetID ids.ID, expectedIdx uint64) {
 	require := require.New(t)
 
 	addressDB := prefixdb.New(sourceAddress[:], db)
 	assetDB := prefixdb.New(assetID[:], addressDB)
 
-	expectedIdxBytes := make([]byte, wrappers.LongLen)
-	binary.BigEndian.PutUint64(expectedIdxBytes, expectedIdx)
-
+	expectedIdxBytes := database.PackUInt64(expectedIdx)
```

Comment:
We should probably make this change in the actual indexing code as well - but I didn't want to expand the real code changes.